### DISCUSSION TITLE
Starter TIE processing is time consuming

### DISCUSSION THREAD

**Discussion by MihailTsybakov** [OpenRadioss/OpenRadioss#3006] [Q&A]
Hi everyone!
This question is an addition to https://github.com/orgs/OpenRadioss/discussions/2808.
Our experiments have shown that the major addition to the starter time consumption is induced by TIE constraint in the model. 
Is there any explanation on this, and any way to fix? We suspect the Tie searching algorithms to be the reason.
For example, for 3M nodes model, if the tie interface contains 50k nodes, starter takes 8 minutes without Tie and 34 minutes with it.

**UPD**
I found out that stated issue is only present if 2021 version is used. With 2017/2020 version of Radioss, starter finished within 20-30 seconds instead of several hours.

Thanks in advance!

**Comment by PaulAltair** [comment]
Hi, you mention versions (2021/2017 etc), are you using Altair Radioss? you should send example to their technical support

Or if you can share a small example here we may test where the slowdown is coming from, I have not experienced tied interface slowing a model like this
